src/integrationtest/java/com/ericsson/ei/frontend/CommonSteps.java

```diff
@@ -67,6 +67,11 @@
 
     private static final Logger LOGGER = LoggerFactory.getLogger(CommonSteps.class);
 
+    @Before
+    public void beforeAllScenarios() {
+        baseURL = String.join(":", frontendHost, String.valueOf(frontendPort));
```

Comment:
Once you split frontEndHost (Note front'E'nd) you may build this string using String.format("%s://%s:%d", args.... x3 )